package.json

```diff
@@ -17,7 +17,7 @@
     "build:sol": "truffle compile all",
     "strip": "babel-node src/tools/strip_contracts.js src/contracts",
     "clean": "rm -fr ./src/contracts/* ./dist/*",
-    "test:units": "mocha",
+    "test:units": "mocha --config=./test/mocharc.json --recursive",
```

Comment:
Not sure why we need this

Reply:
this is the result of migration from old mocha. it doesn't work without this.

Reply:
[https://github.com/ambrosus/ambrosus-node-contracts/pull/296/commits/18d2094b418c2516ad52b1c8a12831206558dac2](url)